### ISSUE TITLE
Autocrafting - Process crafting using Soldering Station fails on Speed Upgrade

### ISSUE THREAD

**Issue by BrewAndCode** [raoulvdberge/refinedstorage#693]
#### Issue description:

Configuration:
4 Auto Crafters handling pattern recipes
1 Auto Crafter handling process recipes w/Soldering Station
1 Importer on bottom of Soldering station to remove completed items

Full Crafting chain for Speed Upgrade module supported.

When I order a speed upgrade it:

Process crafts glass  (furnace->glass)
Process crafts all soldering station components for blank upgrade chip
Pattern crafts sugar.
removes redstone from inventory
removes sugar from inventory
removes 'blank' upgrade chips from inventory

At this point the crafting monitor says it's waiting on the final step, the actual processing of the speed upgrades.

This is done via the process crafter hooked up to the soldering station which successfully worked to create all the components needed for the blank upgrade chip.

however the items never make it into the Soldering station.  (This is on a SMP) I had the inventory of the soldering station open while another player performed the craft.  I watched all of the prerequisite components go into the soldering station and out again.  The speed parts never entered the soldering station.

I repeated the process with all of the prerequisite items already crafted in inventory.  10 blanks, 10 sugar, 10 redstone.

The results were the same.  the items were removed from inventory, the crafting monitor hung at that point, and the items never entered the soldering station.

This seems to be an issue with the method of insertion failing.

#### What you expected to happen:

Items enter Soldering station, craft, are inserted into inventory from Importer.

#### Steps to reproduce:

Have torn down/reset system and same result. happens.

1.
2.
3.
...

#### Version (Make sure you are on the latest version before reporting):

- Minecraft: 1.10.2
- Forge: forge-1.10.2-12.18.2.2151-universal.jar
- Refined Storage: 1.2.6

Does this issue occur on a server? [yes/no] yes

#### If a (crash)log is relevant for this issue, link it here:

no crash log, no logging of any event related to this.

[pastebin/gist/etc link here]

**Comment by BrewAndCode** [comment]
The more I look at this the more I think the problem revolves around the fact that neither the blank upgrades nor the speed upgrades stack -- but it's probably trying to push 'em all into the crafter at once.

There's no reason that those upgrade modules SHOULDN'T stack.  They don't have a configuration, nor do they have DMG.  I haven't dug into any metadata yet -- so I'm not sure if you're using metadata to store temp information when the speed module is installed.

either way -- just a direction to look.

**Comment by way2muchnoise** [comment]
Then not stacking shouldn't be the issue.

**Comment by Moogrok** [comment]
I have the same issue.
This did work in version 1.2.4, but when i upgraded to 1.2.6 this issue occured.
I did some trouble shooting.
When I autocraft it stops just like in the report. The upgrades that was created is voided and is not in the system. 
When I noticed this I performed the following:
I shift click (I think it was 3 of each) sugar, upgrade and redstone into the solderer the first speed upgrade is created. 
But when the second should start it only processes the upgrade, the suger and redston stays in their slots.  The upgrade that is processed is voided once processing is compleete.

The thing that is strange is that manually I can get the speed upgrades to be crafted if i add the resources for one speed upgrade at the time. 
But when autocrafting none at all gets created.

I hope this helps!
